### ISSUE TITLE
logLevel command does not support HA: SocketException

### ISSUE THREAD

**Issue by jiacheliu3** [Alluxio/alluxio#12889]
**Alluxio Version:**
2.5.0-SNAPSHOT-c1694353b4051be8be675903bba1080e39e8aa67

**Describe the bug**
`alluxio logLevel --target master` will only send request to the local master, not the current leader.

The masters are in HA:
```
alluxio.master.hostname=masters-1
alluxio.master.embedded.journal.addresses=masters-1:19200,masters-2:19200
```

Running command on `master-1`, whereas the leader is master-2. The local master is standby so there's no REST service running. So the command results in an error.
```
masters-1$ bin/alluxio fs leader
masters-2
masters-1$ bin/alluxio logLevel --target master --logName alluxio --level DEBUG
Failed to set log level:
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:210)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:137)
	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:153)
	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:280)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:138)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:56)
	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:259)
	at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:163)
	at org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:157)
	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:273)
	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:125)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:272)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at alluxio.util.network.HttpUtils.post(HttpUtils.java:70)
	at alluxio.cli.LogLevel.setLogLevel(LogLevel.java:195)
	at alluxio.cli.LogLevel.logLevel(LogLevel.java:116)
	at alluxio.cli.LogLevel.main(LogLevel.java:210)

```

In `logs/user/user_blah.log`
```
2021-02-19 13:11:25,829 INFO  RetryExec - I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://master-2:19999: The target server failed to respond
2021-02-19 13:11:25,836 INFO  RetryExec - Retrying request to {}->http://master-2:19999
```

**To Reproduce**
Steps to reproduce the behavior (as minimally and precisely as possible)

**Expected behavior**
`logLevel` should be able to at least set the log level on the current leader, no matter where the command is executed.

**Urgency**
Describe the impact and urgency of the bug.

**Additional context**

**Comment by JySongWithZhangCe** [comment]
@jiacheliu3 This bug related with https://github.com/Alluxio/alluxio/issues/12447

**Comment by apc999** [comment]
@jiacheliu3  has this issue been resolved in our PR #12886?

**Comment by jiacheliu3** [comment]
Resolved in https://github.com/Alluxio/alluxio/pull/12874
